### PR TITLE
Remove character limit on !htmlbox

### PR DESCRIPTION
**Pull request by SolarisFox**

### CODE REVIEW

File: command-parser.js [hunk 1]
```diff
@@ -309,7 +309,7 @@ class CommandContext {
 				connection.popup("Your message can't be blank.");
 				return false;
 			}
-			if (message.length > MAX_MESSAGE_LENGTH && !user.can('ignorelimits')) {
+			if (message.length > MAX_MESSAGE_LENGTH && !user.can('ignorelimits') && message.substr(0, 8) !== '!htmlbox') {
```
Comment (Zarel) [inline command-parser.js]: Does this work?

This doesn't seem like the right level of abstraction...

Comment (SolarisFox) [inline command-parser.js]: I opened a local server and it seemed to work. If you want I can do a more thorough test when I get home.

Comment (SolarisFox) [inline command-parser.js]: Screenshot showing logs of what the involved variables are:
http://puu.sh/okUC8.png

**Comment by Zarel** [comment]
Would give htmlbox a different, higher limit, but definitely not "no limit"

**Comment by SolarisFox** [comment]
Should a limit be specifically for htmlbox or should we add a check into `canHTML`?

**Pull request merged**